### PR TITLE
Update README files of RAI QA RAG and non-RAG evaluation flows

### PR DESCRIPTION
**Pull request by qusongms**
# Description
add a section in README files of RAI QA RAG and non-RAG evaluation flows to showcase how to run and evaluate a main chat flow. 

# All Promptflow Contribution checklist:
- [ x] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/flows/evaluation/eval-qna-non-rag/README.md [hunk 1]
```diff
@@ -64,4 +64,33 @@ pf flow test --flow . --inputs metrics="ada_similarity,gpt_fluency,f1_score" que
...
+You can use this evaluation flow to measure the quality and safety of your flow responses.
+
+After the chat flow run is finished, you can this evaluation flow to the run:
+```bash
+pf run create --flow . --data data.jsonl --column-mapping groundtruth='${data.ground_truth}' answer='${run.outputs.answer}' context='{${data.context}}' question='${data.question}' metrics='gpt_groundedness,f1_score'  --run basic_chat_run --stream --name evaluation_qa
+```
+Please note the flow run to be evaluated is specified with `--run basic_chat_run`. Also same as previous run, the evaluation run is named with `--name evaluation_qa`.
+You can view the evaluation run details with:
+```bash
+pf run show-details -n evaluation_qa
```
Comment (zhengfeiwang) [inline examples/flows/evaluation/eval-qna-non-rag/README.md:94]:
```suggestion
# pf run show-details -n evaluation_qa
```

File: examples/flows/evaluation/eval-qna-non-rag/README.md [hunk 2]
```diff
@@ -64,4 +64,33 @@ pf flow test --flow . --inputs metrics="ada_similarity,gpt_fluency,f1_score" que
...
 ```
 You can also skip providing `column-mapping` if provided data has same column name as the flow.
-Reference [here](https://aka.ms/pf/column-mapping) for default behavior when `column-mapping` not provided in CLI.
\ No newline at end of file
+Reference [here](https://aka.ms/pf/column-mapping) for default behavior when `column-mapping` not provided in CLI.
+
+## 3. Run and Evaluate your flow with this Q&A evaluation flow
+After you develop your flow, you may want to run and evaluate it with this evaluation flow. 
+
+Here we use the flow [basic_chat](https://github.com/microsoft/promptflow/tree/main/examples/flows/chat/basic-chat) as the flow to evaluate. It is a flow demonstrating how to create a chatbot with LLM.  The chatbot can remember previous interactions and use the conversation history to generate next message, given a question.
```
Comment (zhengfeiwang) [inline examples/flows/evaluation/eval-qna-non-rag/README.md:72]:
```suggestion
Here we use the flow [basic_chat](../../chat/basic-chat/) as the flow to evaluate. It is a flow demonstrating how to create a chatbot with LLM.  The chatbot can remember previous interactions and use the conversation history to generate next message, given a question. 
```

File: examples/flows/evaluation/eval-qna-rag-metrics/README.md [hunk 3]
```diff
@@ -53,3 +53,32 @@ pf run create --flow . --data ./data.jsonl --column-mapping question='${data.que
 ```
 You can also skip providing `column-mapping` if provided data has same column name as the flow.
 Reference [here](https://aka.ms/pf/column-mapping) for default behavior when `column-mapping` not provided in CLI.
+
+## 3. Run and Evaluate your flow with this Q&A RAG evaluation flow
+After you develop your flow, you may want to run and evaluate it with this evaluation flow. 
+
+Here we use the flow [basic_chat](https://github.com/microsoft/promptflow/tree/main/examples/flows/chat/basic-chat) as the main flow to evaluate. It is a flow demonstrating how to create a chatbot with LLM. The chatbot can remember previous interactions and use the conversation history to generate next message, given a question.
```
Comment (zhengfeiwang) [inline examples/flows/evaluation/eval-qna-rag-metrics/README.md:60]:
```suggestion
Here we use the flow [basic_chat](../../chat/basic-chat/) as the main flow to evaluate. It is a flow demonstrating how to create a chatbot with LLM. The chatbot can remember previous interactions and use the conversation history to generate next message, given a question. 
```

File: examples/flows/evaluation/eval-qna-non-rag/README.md (hunk 2 above)
Comment (zhengfeiwang) [reply examples/flows/evaluation/eval-qna-non-rag/README.md:72]: I prefer local path than url, feel free to apply or ignore (same below)

Comment (qusongms) [reply examples/flows/evaluation/eval-qna-non-rag/README.md]: Changed url to local path.
Thanks!

File: examples/flows/evaluation/eval-qna-rag-metrics/README.md (hunk 3 above)
Comment (qusongms) [reply examples/flows/evaluation/eval-qna-rag-metrics/README.md]: Changed url to local path.
Thanks!

**Review by zhengfeiwang — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by crazygao — APPROVED** [review]

**Pull request merged**